index.html

```diff
@@ -584,13 +592,18 @@ <h2>Terminology</h2>
 this document to other specifications. This specification decouples the
 [=identity provider=] concept into two distinct concepts: the [=issuer=]
 and the [=holder=].
+        </dd>
+        <dt><dfn class="export" data-lt="issuee|issuee's">issuee</dfn></dt>
+        <dd>
+A role an [=entity=] performs when it receives a [=verifiable credential=] from
+the [=issuer=].
         </dd>
         <dt><dfn class="export" data-lt="issuers|issuer's">issuer</dfn></dt>
         <dd>
 A role an [=entity=] can perform by asserting [=claims=] about one or
 more [=subjects=], creating a [=verifiable credential=] from these
```

Comment:
As above, does this echo/repeat lines 369-372? 
Does the `data-lt` attribute value (now singular, plural, and possessive) need to include the (singular) string within the `<dfn>`?
```suggestion
        <dt><dfn class="export" data-lt="issuer|issuers|issuer's">issuer</dfn></dt>
        <dd>
A role an [=entity=] can perform by asserting [=claims=] about one or
more [=subjects=], creating a [=verifiable credential=] from these
```

Reply:
I have not made any edits to this text, so this should be a separate PR on the current CR.